GdscBackend/Controllers/v1/AuthenticationController.cs

```diff
@@ -0,0 +1,88 @@
+using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+using gdsc_web_backend.Authentication;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.IdentityModel.Tokens;
+
+namespace gdsc_web_backend.Controllers.v1
+{
+    [Route("api/v1/auth")]
+    [ApiController]
+    public class AuthenticationController : ControllerBase
+    {
+        private readonly UserManager<User> userModel;
+        private readonly IConfiguration _configuration;
+
+        public AuthenticationController(UserManager<User> userModel, IConfiguration configuration)
+        {
+            this.userModel = userModel;
```

Comment:
"this" not needed

Reply:
este nevoie de acel this pentru ca programul nu va stii la care userModel atribuie acea valoare de userModel